docs/docsite/rst/community/reporting_bugs_and_features.rst

```diff
@@ -15,36 +16,45 @@ Reporting a bug
+- affecting multiple plugins
+- a plugin that remained in the ansible/ansible repo
+- the overall functioning of Ansible
 
-Do not open issues for "how do I do this" type questions.  These are great topics for community chat channels or a mailing list, where things are likely to be more of a discussion.
+How to write a good bug report
+------------------------------
 
-If you find a bug, open the issue yourself to ensure we have a record of it. Do not rely on someone else in the community to file the bug report for you. We have created an issue template, which saves time and helps us help everyone with their issues more quickly. Please fill it out as completely and as accurately as possible:
+If you find a bug, open an issue using the `issue template <https://github.com/ansible/ansible/issues/new?assignees=&labels=&template=bug_report.yml>`. Detail what you've tried, why you think this is a bug, and what component to use. Fill it out as completely and as accurately as possible, include:
 
-  * Include the Ansible version
-  * Include any relevant configuration
-  * Include the exact commands or tasks you are running
-  * Describe the behavior you expected
-  * Provide steps to reproduce the bug
-    * Use minimal well-reduced and well-commented examples, not your entire production playbook
-    * When sharing YAML in playbooks, preserve the formatting by using `code blocks  <https://help.github.com/articles/creating-and-highlighting-code-blocks/>`_.
-  * Document the behavior you got
-  * Include output where possible
-  * For multiple-file content, use gist.github.com, which is more durable than pastebin content
+  * your Ansible version
+  * any relevant configurations
+  * the exact commands or tasks you are running
+  * the expected behavior
+  * the steps to reproduce the bug
+    * Use a minimal reproducible example and comments describing examples
+    * Preserve formatting using `code blocks  <https://help.github.com/articles/creating-and-highlighting-code-blocks/>`_ when sharing YAML in playbooks.
+  * the behavior you currently see
+  * output where possible
 
-If you are not sure whether a bug is in ansible-core or in a collection, you can report the behavior on the :ref:`mailing list or community chat channel first <communication>`.
+For multiple-file content, use gist.github.com, which is more durable than pastebin content.
 
 .. _request_features:
 
 Requesting a feature
 ====================
 
 Before you request a feature, check what is :ref:`planned for future Ansible Releases <roadmaps>`.
-The best way to get a feature into Ansible is to :ref:`submit a pull request <community_pull_requests>`, either against ansible-core or against a collection. See also :ref:`ansible_collection_merge_requirements`.
+To get your feature into Ansible, to :ref:`submit a pull request <community_pull_requests>`, either against ansible-core or against a collection. See also :ref:`ansible_collection_merge_requirements`.
```

Comment:
Another thought: Can you add something like this?

`For existing pull requests for new features, refer to open list of open feature-related pull requests (or similar text` to address "How to see if a PR is already in flight"

Reply:
https://github.com/ansible/ansible/issues?q=is%3Aissue+is%3Aopen+label%3Afeature ?